runasadministrator_windows_test.go

```diff
@@ -0,0 +1,177 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestRunAsAdministratorDisabled(t *testing.T) {
+	defer setup(t)()
+	if config.RunTasksAsCurrentUser {
+		t.Skip("Skipping since running as current user...")
+	}
+	payload := GenericWorkerPayload{
+		Command: []string{
+			`whoami /groups`,
+			`whoami /groups | C:\Windows\System32\find.exe "S-1-16-12288" > nul`,
```

Comment:
Could you add a comment which explains why this security id is being used?  I did some searching and found [this](https://support.microsoft.com/en-us/help/243330/well-known-security-identifiers-in-windows-operating-systems).  I'm not too sure how these identifiers work, but would it be worth while testing for the absence of "S-1-5-32-544" instead -- or is this security id the one that UAC would give to a ran-as-admin user?

Reply:
Yes, I'll do that, nice spot! :+1: